lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -258,74 +258,116 @@ def _get_converged_core_config(service_mconfig: object) -> bool:
 
-    Returns: slice service type value
+    Returns:
+        slice service type value.
     """
+    enable_default_service_slice_type = get_service_config_value(
+        'mme', 'default_slice_service_type', None,
+    )
+
+    if enable_default_service_slice_type is not None:
+        return enable_default_service_slice_type
+
     return service_mconfig.default_slice_service_type or DEFAULT_NGAP_S_NSSAI_SST
 
 
 def _get_default_slice_differentiator_type_config(service_mconfig: object) -> str:
-    """
-    Retrieves default_slice_differentiator config value,If it does not exist
-    it defaults to 0xffffff.
+    """Retrieve default_slice_differentiator config value,If it does not exist it defaults to 0xffffff
+
     Args:
-        service_mconfig:
+        service_mconfig: This is a configuration placeholder for mme.
 
-    Returns: slice differentiator config value
+    Returns:
+        slice differentiator config value.
     """
+    enable_default_slice_differentiator_type = get_service_config_value(
+        'mme', 'default_slice_differentiator', None,
+    )
+
+    if enable_default_slice_differentiator_type is not None:
+        return enable_default_slice_differentiator_type
+
     return service_mconfig.default_slice_differentiator or DEFAULT_NGAP_S_NSSAI_SD
 
 
 def _get_amf_name_config(service_mconfig: object) -> str:
-    """
-    Retrieves amf_name config value,If it does not exist
-    it defaults to Default Name AMF_1.
+    """Retrieve amf_name config value,If it does not exist, it defaults to Default Name AMF_1
+
     Args:
-        service_mconfig:
+        service_mconfig: This is a configuration placeholder for mme.
 
-    Returns: amf name string
+    Returns:
+        amf name string.
     """
+    enable_amf_name_config = get_service_config_value(
+        'mme', 'amf_name', None,
+    )
+
+    if enable_amf_name_config is not None:
+        return enable_amf_name_config
+
     return service_mconfig.amf_name or DEFAULT_NGAP_AMF_NAME
 
 
 def _get_amf_region_id(service_mconfig: object) -> str:
-    """
-    Retrieves amf_region_id config value,If it does not exist
-    it defaults to Default value of 1
+    """Retrieve amf_region_id config value, If it does not exist it defaults to Default value of 1
+
     Args:
-        service_mconfig:
+        service_mconfig: This is a configuration placeholder for mme.
 
-    Returns: amf region id
+    Returns:
+        amf region id.
     """
+    enable_amf_region_id = get_service_config_value(
+        'mme', 'amf_region_id', None,
+    )
+
+    if enable_amf_region_id is not None:
+        return enable_amf_region_id
+
     return service_mconfig.amf_region_id or DEFAULT_NGAP_AMF_REGION_ID
 
 
 def _get_amf_set_id(service_mconfig: object) -> str:
-    """
-    Retrieves amf_set_id config value,If it does not exist
-    it defaults to Default String 1
+    """Retrieve amf_set_id config value, If it does not exist it defaults to Default String 1
```

Comment:
punctuation and capitalization issues (`,If` , missing `.` at the end of the sentence).

Reply:
Comment addressed.